dependency_support/pip_requirements.in

```diff
@@ -13,6 +13,11 @@ pyyaml==6.0.1
 # We build most of z3 ourselves but building python is really complicated. Just
 # use pypi version
 z3-solver==4.14.0.0
+pytest==8.2.2
+cocotb==1.9.0
```

Comment:
FYI, internally we are on `v1.7.1`, so if there is anything that's specific to 1.9.0, will need to upgrade the internal copy of cocotb first; as I commented elsewhere it might be better to untie this and the associated python tests from this PR and import this separately.